how-to/use_terraform.rst

```diff
@@ -0,0 +1,133 @@
+JAAS: Using Terraform
+=====================
+
+Introduction
+------------
+
+In this how-to we will be showing you how to use Terraform with JAAS.
+
+Prerequisites
+-------------
+
+For this how-to you will need the following:
+
+- An identity provider that can be used to create OAuth2 client credentials
+- Client credentials (`client_id` and `client_secret`) generated by the above identity provider.
```

Comment:
As a difference with Markdown, in RST we should use double-ticks instead of one (e.g.,  ``client_id``). Please apply this to other instances as well.